src/GovUk.Education.ExploreEducationStatistics.Public.Data.Scripts/Commands/SeedDataCommand.cs

```diff
@@ -326,7 +340,12 @@ public async Task Generate()
                         .OrderBy(row => row.Label)
                         .Select(row => row.Label)
                         .ToList(),
-                    GeographicLevels = ListGeographicLevels(allowedColumns)
+                    GeographicLevels = geographicLevels
+                },
+                GeographicLevelMeta = new GeographicLevelMeta
+                {
+                    DataSetVersionId = _seed.DataSetVersionId,
+                    Levels = geographicLevels
```

Comment:
Doing this as part of `CreateDataSetMeta` would involve changing `GeographicLevelMeta` to not be required on `DataSetVersion`.

This isn't the worst thing, but means that `GeographicLevelMeta` would need to be nullable. We'd have to handle nullability in any consuming code, so it'd be better to avoid it if we can.

Given that we should know what the geographic levels are as soon as we construct the version and its meta summary (like in our seeding code), it doesn't seem like we need to take the hit on nullability here.

I've left it as-is for now unless you can see any issue with the above rationale.

Reply:
Makes sense!

Although, now I'm questioning why we didn't make `LocationMetas` etc. required. Because we will know those at construction too right?